packages/expo-manifests/android/src/main/java/expo/modules/manifests/core/NewManifest.kt

```diff
@@ -15,11 +15,7 @@ class NewManifest(json: JSONObject) : Manifest(json) {
   @Throws(JSONException::class)
   fun getID(): String = json.require("id")
 
-  /**
-   * Incorrect for now until we figure out how to get this in the new manifest format.
-   */
```

Comment:
I think this comment is still partly relevant until we read the stable legacy ID from "extra" if possible

Reply:
The idea is that we won't ever read the stable legacy ID in new manifests (or get any identifiers from `extra.expoClient` for that matter). That way we don't ever accidentally become dependent on it, and also don't ever use any identifiers other than scopeKey and projectId, and can maybe remove full name in the future after a reasonable amount of time and a thorough check.